### PR TITLE
Update metadata for Learn docs: Calendar

### PR DESCRIPTION
**Pull request by Linda-Editor**
Updated meta title and description per SEO guidelines. Sometimes updated minor errors I happened to notice on a page.

### CODE REVIEW

File: concepts/outlook-change-notifications-overview.md [hunk 1]
```diff
@@ -191,8 +197,12 @@ Content-type: application/json
 ```
 
 #### Response
-The following is an example of the response. 
->**Note:** The response object shown here might be shortened for readability.
+
+The following is an example of the response.
+
+> [!NOTE]
```
Comment (Lauragra) [inline concepts/outlook-change-notifications-overview.md:203]: For this standard boilerplate in particular, we opt to use the less "intrusive" markdown note style (>). Can we restore that? For the other places where you apply the other alert syntax, just to verify, did you verify the content flow and such on the review site?

Comment (Linda-Editor) [reply concepts/outlook-change-notifications-overview.md:203]: So are you saying that you use the > note style in code samples (request/response) only? Most of my changes are behind the scenes (in the metadata), but places where I do make visible changes I do review them.

Comment (Linda-Editor) [reply concepts/outlook-change-notifications-overview.md:203]: I changed back the 3 instances that were in the Response sections.

File: concepts/outlook-share-or-delegate-calendar.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Share or delegate a calendar in Outlook"
-description: "In Outlook, a calendar owner can share the calendar with another user, or delegate another user to manage meetings in the owner's primary calendar."
+description: "In Outlook, as a calendar owner, you can share the calendar with another user or delegate another user to manage meetings in your primary calendar."
```
Comment (Lauragra) [inline concepts/outlook-share-or-delegate-calendar.md:3]: In our developer content, "you" is the developer target audience, whereas here, "you" is referring to the end user of an Outlook calendar. Can we update this description to be developer focused? Something like "Learn how to use the Microsoft Graph calendar API to manage calendar sharing and delegation."

File: concepts/choose-online-meeting-api.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Choose an API in Microsoft Graph to create and join online meetings"
-description: "You'll have the flexibility to create a meeting that takes place in the future, or instantaneously"
+description: "Consider when to use the calendar API event resource or the communications API onlineMeeting resource for Teams and Skype meetings in Outlook calendar."
```
[the remark this answers is not in the document]
Comment (Linda-Editor) [reply concepts/choose-online-meeting-api.md]: Changed back per guidance from Laura.

**Pull request merged**